Makefile.am

```diff
@@ -452,12 +454,22 @@ libwx_new_la_LIBADD = $(WX_LIBS)
 
 # auxiliary executables
 
+md5sum_SOURCES = \
+    md5.cpp \
```

Comment:
There should be `getopt.cpp` here, without it the utility doesn't link when using autotools.